oracle/metrics_test.go

```diff
@@ -57,7 +57,7 @@ func (s *OracleTestSuite) TestMetrics() {
 		}
 	}()
 
-	metrics.On("SetSlinkyBuildInfo").Return()
+	metrics.On("SetConnectBuildInfo").Return()
```

Comment:
if you want, the changes on main generates mocks with typesafe expects. so if you merge main and do

`make mocks`

you can change this line to:

```go
metrics.EXPECT().SetConnectBuildInfo().Return()
```

Reply:
excellent